cactusbot/api.py

```diff
@@ -26,7 +26,20 @@ def __init__(self, token, password, url=URL, auth_token="", **kwargs):
         self.token = token
         self.auth_token = auth_token
         self.password = password
-        self.URL = url
+        self.url = url
+
+        self.buckets = {
```

Comment:
Thanks! Much cleaner than what we had before - it's nice to have things grouped together, too. :smile: